load/PreValidate.py

```diff
@@ -71,7 +71,7 @@ def validateDBPETL(self, s3Client, location, directoryName, fullPath):
 				self.validateLPTS(result)
 				resultList = [result]
 
-		print("PreValidate.validateDBPETL. returning resultList[0] %s, messages: %s" % (resultList[0], self.messages))
+		print("PreValidate.validateDBPETL. returning resultList[0] %s, messages: %s" % (resultList, self.messages))
```

Comment:
when filesets are specified, the resultList is empty, I think.. I think this line should be commented out for now

Reply:
@bradflood , I have also removed it. I think it is no longer necessary.